package.json

```diff
@@ -60,7 +60,8 @@
     "updtr": "^2.0.0"
   },
   "dependencies": {
-    "prop-types": "^15.5.10"
+    "prop-types": "^15.5.10",
+    "@react-native-community/viewpager": "^3.1.0",
```

Comment:
@douglashsilva Can you remove the comma from the end of this line?

Reply:
Yes